exec-analysis/src/main/java/com/capitalone/dashboard/exec/collector/DefaultDataCollector.java

```diff
@@ -59,4 +69,26 @@
             });
         return rowMap;
     }
+
+    protected Dataset<Row> getDataRowsForPipelines() {
+        Dataset<Row> dataRows = null;
+        String tempQuery = query;
+        String[] envName = {"prod"}; //If we have more envName, update in a string with comma separated (Ex:{"prod","Production"})
```

Comment:
Can this be loaded from collector properties?

Reply:
Yes, We have updated envNames to application.properties as List. In exec analysis, properties file have to update with portfolio.pipelineEnvName=prod,production